src/app/home/home/home.component.ts

```diff
@@ -168,19 +169,21 @@ export class HomeComponent implements OnInit, AfterViewInit {
 
   ngAfterViewInit() {
     for(let i=0; i<this.sectionList.length;i++) {
-      if(this.sectionList[i]['section'] == 'section_0' 
+      if(this.sectionList[i]['section'] == 'section_0'
       || this.sectionList[i]['section'] == 'section_1'
       || this.sectionList[i]['section'] == 'section_2') {
-        this.sectionList[i]['isVisible'] = true;        
-      } 
+        this.sectionList[i]['isVisible'] = true;
+      }
     }
   }
-  
+
   getEnrollmentData(){
-    let enrollData: any = localStorage.getItem('enrollmentData')
-    enrollData = JSON.parse(enrollData)
-    if (enrollData && enrollData.courses && enrollData.courses.length > 0) {
-      this.isKPPanelenabled = true
+    this.enrollData = localStorage.getItem('enrollmentData')
+    if (this.enrollData) {
+      this.enrollData = JSON.parse(this.enrollData)
+      if (this.enrollData && this.enrollData.courses && this.enrollData.courses.length > 0) {
```

Comment:
can we use "this.enrollData.courses.length" only instead of > 0

Reply:
@vishnubansaltarento fixed all the mentioned comments